modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/moth.dm

```diff
@@ -1,23 +1,8 @@
 /datum/species/moth
 	mutant_bodyparts = list()
-	default_mutant_bodyparts = list(
-		"fluff" = "None",
-		"wings" = ACC_RANDOM,
-		"moth_antennae" = ACC_RANDOM
-	)
-	species_traits = list(
-		LIPS,
-		NOEYESPRITES,
-		HAS_FLESH,
-		HAS_BONE,
-		HAS_MARKINGS,
-		MUTCOLORS
-	)
-	inherent_traits = list(
-		TRAIT_ADVANCEDTOOLUSER,
-		TRAIT_CAN_STRIP,
-		TRAIT_CAN_USE_FLIGHT_POTION,
-	)
+	default_mutant_bodyparts = list("fluff" = "None","wings" = ACC_RANDOM,"moth_antennae" = ACC_RANDOM)
+	species_traits = list(LIPS,HAS_FLESH,NOEYESPRITES,HAS_BONE,HAS_MARKINGS,MUTCOLORS)
```

Comment:
Your branch is not in sync.

Reply:
Odd. Heck it. Let's gooooo!

Reply:
Seems to be a thing, but doesn't seem to work the colors properly. Everything goin' tan. Which is what I'm tryin' ta fix.
I'll tweak as needed but things seem to be rockin' it on my end.
![image](https://user-images.githubusercontent.com/84706993/163665897-e6472f65-bda2-4567-bb57-9953c9a1c6df.png)